Sources/Rendering/Core/AxesActor/index.js

```diff
@@ -117,24 +133,48 @@ function vtkAxesActor(publicAPI, model) {
     return false;
   };
 
-  publicAPI.setXAxisColor = (c) => {
-    if (setXAxisColor(c)) {
+  publicAPI.setXConfig = (c) => {
+    if (setXConfig(c)) {
+      _debouncedUpdate();
+      return true;
+    }
+    return false;
+  };
```

Comment:
there is now better than overriding setters:
```
model._onXConfigChanged = _debouncedUpdate;
model._onYConfigChanged = _debouncedUpdate;
model._onZConfigChanged = _debouncedUpdate;
```
they will be automatically called when `setXConfig()` is change the model value (see `macros.js:findSetter:getSetter::setter`)